R/visr.R

```diff
@@ -560,3 +562,11 @@ visr.tidycuminc <- function(x = NULL
   gg
 }
 
+
+.construct_strata_label <- function(estimate_args, sep = ", ") {
+  if (is.null(estimate_args[["strata"]])) return(" ")
```

Comment:
Do we expect this to be NULL?
If we estimate using the formulat ~ 1 we add an artifical stratum "Overall" to facilitate downsteam processing.

Reply:
i updated the structure, so this line no longer appears. In this setting, the strata arg was NULL